pytest_reportportal/service.py

```diff
@@ -630,12 +630,20 @@ def get_marker_value(item, keyword):
             get_marker = getattr(item, "get_closest_marker")
         except AttributeError:
             get_marker = getattr(item, "get_marker")
-        attributes = [{"value": get_marker_value(item, k)}
+        raw_attributes = [get_marker_value(item, k)
                       for k in item.keywords if get_marker(k) is not None
                       and k not in self.ignored_attributes]
 
-        attributes.extend([{"value": tag} for tag in
-                           item.session.config.getini('rp_tests_attributes')])
+        raw_attributes.extend([tag for tag in item.session.config.getini('rp_tests_attributes')])
+
+        attributes = []
+        for attr in raw_attributes:
```

Comment:
Duplicates existing code in the get_launch_attributes function.

Reply:
Duplicated code was moved to a new static function